packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md

```diff
@@ -2,138 +2,98 @@
+**Q: My app is now in the OIN, what is the user experience for a joint customer administrator that wants to set up single sign-on and provisioning for my app in the Okta interface?**
 
-**How does data validation work with SCIM provisioning? For example, if my app requires the phone number in a specific format, how do I ensure that Okta passes the attribute in that format? If a data validation error issue occurs how does error reporting work?**
+Okta creates unique SAML configuration documentation for each application in the OIN, so each will be different. For a sample configuration, see our instructions for [How to Configure SAML 2.0 for Salesforce](http://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html). Also, you can search online for videos demonstrating a step-by-step walk through of all the steps an IT administrator would take to configure single sign-on and provisioning for an app.
 
-The SCIM spec specifies valid data formats for a given user profile attribute, however Okta does not rigorously validate that the customer has inputted values meeting those requirements to preserve flexibility.
+### SCIM
 
-Therefore, data validation should be handled by your app's SCIM Server. In other words, when Okta provisions user profile to your app, it should check that the data is valid according to their special requirements. Error messages sent in the response from your app are surfaced to the Okta administrator through alerts and tasks in the Okta interface. You should also specify your data requirements in your config guide.
+SCIM FAQs are covered in the [SCIM - Frequently Asked Questions](/docs/concepts/scim/faqs/) document.
 
-**How much filtering support is needed?**
+### WS-Fed
 
-The filtering capabilities in the SCIM protocol are pretty broad, but the common filtering use case with Okta is quite narrow. Okta determines if a newly created user already exists in your app based on a matching identifier. This means the `eq` (equals) operator is all you really need to support.
+**Q: My app currently supports WS-FED for single sign-on. Can I use the App Wizard?**
 
-Note that Okta only supports filtering through the `eq` operator on the SCIM `userName` attribute on the SCIM server side. However, Okta can use any `AppUser` attribute on the Okta side for passing into the `eq` operator. Typically this is `appuser.userName`, but `appuser.email` or `appuser.randomAttribute` can also be used.
+The Okta App Wizard only supports SAML 2.0 for federated single sign-on. If your app supports WS-Fed, create a [WS-Fed Template App](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Configuring_WS_Federation). Afterwards, the template application you created can be used only within your account. In order to promote your template app to the Okta Integration Network, please email a screenshot with the details of the configured app to <developers@okta.com>, with your app name in the subject line.
```

Comment:
The Okta App Wizard only supports SAML 2.0 for federated **Single Sign-On**. If your app supports WS-Fed, create a [WS-Fed Template App](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Configuring_WS_Federation). **You can only use the template application that you create within your account**. **To** promote your template app to the Okta Integration Network, **email** a screenshot with the details of the configured app to <developers@okta.com> with your app name in the subject line.

removed the comma before with in the last sentence, too

Reply:
Changed last two lines to read:
"You can only use this created template application within your own account. To promote your template app to the Okta Integration Network, email a screenshot with the details of the configured app to <developers@okta.com> and your app name in the subject line."